README.md

```diff
@@ -1,3 +1,5 @@
+> Forked from https://chrisbsmith.github.io/uptime-kuma-helm
+
```

Comment:
To be honest I think this is not needed to be in the README.
It would clutter up if I add an entry for each contribution or pull request being made.

Reply:
yup, this was only for me. i'll remove